src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,11 @@
 
 public class Application {
```

Comment:
The class 'Application' should have an access modifier, typically 'public' to ensure it is accessible from other packages if needed.